standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5394,6 +5410,476 @@ public void countOpenTxns() throws MetaException {
     }
   }
 
+  private void cleanOldStatsFromPartColStatTable(Map<PartitionInfo, ColumnStatistics> statsPartInfoMap,
+                                                 Connection dbConn) throws SQLException {
+    PreparedStatement preparedStatement = null;
+    int numRows = 0;
+    int maxNumRows = MetastoreConf.getIntVar(conf, ConfVars.DIRECT_SQL_MAX_ELEMENTS_VALUES_CLAUSE);
```

Comment:
Shouldn't you use maxBatchSize (JDBC_MAX_BATCH_SIZE)? I think DIRECT_SQL_MAX_ELEMENTS_VALUES_CLAUSE is for different purposes, at least based on the property description.

Reply:
i think JDBC_MAX_BATCH_SIZE is more appropriate.

Reply:
done